Resources/Prototypes/Datasets/verbs.yml

```diff
@@ -97,9 +97,7 @@
   - coach
   - coil
   - collect
-  - colour
```

Comment:
> PR author believes this will be uncommon enough that the feature is worth implementing with the bug, and can be fixed in a future chat refactor.

i would really rather you just fix the underlying issue than patch it over by removing a few particular problem words.

Reply:
As stated in the github thread, the underlying issue's answer is "Refactor the chat system". You are asking for a system to track formatting tags across messages, along with metadata information such as names, channels and prefix/suffix data. And that would include reformatting all strings to fit within this new systems.

This PR has been in the making for months, but kept at bay since there was supposed to be a chat refactor that ended up never happening. I've made sure to set up this system such that, _if_ and when chat gets refactored, this addition will be simple to convert.